app/services/discourse_translator/base.rb

```diff
@@ -72,14 +72,17 @@ def self.language_supported?(detected_lang)
 
     private
 
-    def self.strip_img_for_detection(detection_text)
+    def self.strip_tags_for_detection(detection_text)
       html_doc = Nokogiri::HTML::DocumentFragment.parse(detection_text)
       html_doc.css("img").remove
+      html_doc.css("a").remove
```

Comment:
how about limiting it to `a.mention` or something? do you want to remove actual text that has links? like `<a href="...">Here's the article I mentioned</a>`